src/components/v5/shared/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import { useMobile } from '~hooks';
+import StepperButton from './partials/StepperButton/StepperButton';
+import { StepperProps } from './types';
+import { STEP_STAGE } from './partials/StepperButton/consts';
+import { MIN_NUMBER_OF_STEPS_WITHOUT_MOBILE_NAVIGATION } from './consts';
+
+const displayName = 'v5.shared.Stepper';
```

Comment:
I think that components inside shared folder don't have `shared` in displayName, please remove it.

Reply:
![image](https://github.com/JoinColony/colonyCDapp/assets/44369011/f2227f45-12f2-441c-a17b-5e39bb2589f4)